### PR TITLE
fix schema

### PR DESCRIPTION
**Pull request by daimaxiaxie**
fix schema

kube client get/list error
```bash
"error":"no kind is registered for the type v1beta1.MachineDeploymentList in scheme \"pkg/runtime/scheme.go:100\""
```

### CODE REVIEW

File: pkg/apis/apis.go [hunk 1]
```diff
@@ -19,13 +19,15 @@ package apis
 import (
 	"k8s.io/apimachinery/pkg/runtime"
 
-	"sigs.k8s.io/karpenter/pkg/apis/v1beta1"
+	clusterv1beta1 "sigs.k8s.io/cluster-api/api/v1beta1"
```
Comment (elmiko) [inline pkg/apis/apis.go:22]: could we call this `capiv1beta1` to keep it consistent with the other usages

File: pkg/apis/apis.go [hunk 2]
```diff
@@ -19,13 +19,15 @@ package apis
...
+	"sigs.k8s.io/karpenter-provider-cluster-api/pkg/apis/v1alpha1"
```
Comment (elmiko) [inline pkg/apis/apis.go:23]: and call this `api`, also to keep consistent with other usages

**Review by elmiko — COMMENTED** [review]
thanks @daimaxiaxie , i'm surprised it doesn't complain when the karpenter apis are removed. this generally looks good, i just have a couple nits about the naming.

File: pkg/apis/apis.go (hunk 1 above)
Comment (daimaxiaxie) [reply pkg/apis/apis.go:22]: ok

File: pkg/apis/apis.go (hunk 2 above)
Comment (daimaxiaxie) [reply pkg/apis/apis.go]: done

**Review by elmiko — COMMENTED** [review]
perfect, thank you!

File: pkg/apis/apis.go (hunk 2 above)
Comment (elmiko) [reply pkg/apis/apis.go]: it's a good point, i was only using `api` during this phase of development, i think we should come up with a better name and refactor the code to use it.

**Pull request merged**